armadactl.md

```diff
@@ -0,0 +1,48 @@
+# Command line tool (armadactl)
+
+Armadactl is a command-line tool used for managing jobs in the Armada workload orchestration system. It provides functionality for creating, updating, and deleting jobs, as well as monitoring job status and resource usage.
+
+This tool is named `armadactl`.
+
+## Syntax
+
+Once Armadactl is successfully installed, you can use it to execute Armada subcommands by running the following command:
+```
+armadactl [subcommand] [flags]
+```
+
+where `subcommand` and `flags` are:
+
+- `command`: Specifies the operation that you want to perform on one or more deployments, for example `create`, `update`, `cancel`, `delete`.
+
+- `flags`: Specifies optional flags.
+
+**Caution**: Flags that you specify from the command line override default values and any corresponding environment variables.
+
+If you need help, run `armadactl --help` from the terminal window.
+
+## Operations
+
+The following table includes short descriptions and the general syntax for all of the armadactl operations:
+
+Operation       | Syntax    |       Description
+-------------------- | -------------------- | --------------------
+`analyze`    | `armadactl analyze [path/to/manifests] [flags]` | The analyze subcommand can be used to analyze a set of manifests and provide a report on their compatibility with a given Kubernetes cluster version.
+`cancel`    | `armadactl cancel [deployment_name] [flags]` | The cancel subcommand can be used to cancel a running Armada deployment.
```

Comment:
Not really sure what you mean by Armada deployment.  Cancel takes either `queue` and `jobSet` or `jobId`.  And it either cancels all the jobs in a queue + jobSet or it cancels an individual job.

Reply:
Hmm I copied the content from the main repository of armada if its incorrect, we might wanna correct this there too
https://github.com/armadaproject/armada/tree/master/cmd/armadactl

Reply:
I see.  I'll see what the team thinks.  If we feel that this is misleading than updating those comments would be useful.